effect/ref.py

```diff
@@ -0,0 +1,60 @@
+from characteristic import attributes
+
+from ._base import Effect
+from ._dispatcher import TypeDispatcher
+from ._sync import sync_performer
+
+
+class ERef(object):
```

Comment:
Why "ERef"?  How about "MutableReference"?

Going with a naming convention because it mirrors the [famous beginner-friendliness](https://wiki.haskell.org/Monad_tutorials_timeline) of the Haskell community's standards or the [plainspoken clarity](https://glyph.twistedmatrix.com/2012/07/simple-made-variadic.html) of Clojure seems misguided :).